runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeOverlay.java

```diff
@@ -129,6 +129,14 @@ else if (itemId == ItemID.CHRONICLE)
 			}
 			charges = plugin.getChronicle();
 		}
+		else if (itemId == ItemID.KHAREDSTS_MEMOIRS)
+		{
+			if (!plugin.isShowKharedstsMemoirs())
+			{
+				return;
+			}
+			charges = plugin.getKharedstsMemoirs();
```

Comment:
This is a varbit (6035), meaning this can be relpaced wiht `charges = client.getVar(Varbits.KKK_MEMOIRS);` (once you add it to that class obviously)

Nothing else but config checks needed

Reply:
I'll have to figure out how these varbits work :)

I put the varbit assignment in the plugin, rather than create an accessor for client, so its consistent with everything else,

```
	@Subscribe
	private void onVarbitChanged(VarbitChanged event)
	{
                // ...
		this.kharedstsMemoirs = client.getVar(Varbits.KHAREDSTS_MEMOIRS_CHARGES);
	}
```

though I'm not sure if that will get called more or less than having it in overlay (render is called every gametick?), i guess it probably doesn't matter.

Reply:
That definitely gets called less; render is called on every frame, nice thinking :P